packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -158,7 +161,10 @@ function AuthOptions({
             title="Verify your email address"
             onClose={onClose}
           />
-          <EmailVerificationSent email={email} />
+          <EmailVerificationSent
+            email={email}
+            onResend={() => onRegister(registrationValues)}
+          />
```

Comment:
I think resending the registration might result in an error as the user already exists. We might start a verification flow. Though I'm not sure tho about the current response.

Reply:
I guess we can also start the verification flow, was unsure at the time what was the needed flow.

Reply:
@sshanzel Added the verification flow now.
Can't test the emails, but response from Kratos is "email_sent"

![Screenshot 2022-08-17 at 15 17 13](https://user-images.githubusercontent.com/554874/185143890-60bf6087-af72-46de-b231-a65d81631078.png)

Reply:
Oh, nice. Thought it was the registration flow getting utilized. Nice. Then this is perfect.